<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lmfitother</class>
 <widget class="QWidget" name="lmfitother">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="group_box">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Orbital Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="2" column="1">
       <widget class="QComboBox" name="symmetrize_combobox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxVisibleItems">
         <number>7</number>
        </property>
        <property name="maxCount">
         <number>7</number>
        </property>
        <item>
         <property name="text">
          <string>No Symmetry</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2-Fold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2-Fold + Mirror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-Fold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3-Fold + Mirror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4-Fold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4-Fold + Mirror</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="polarization_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Polarization:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="symmetrize_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Symmetrization:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="ak_combobox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>|Psi|^2 used to be 'no |A.k|^2'
|A.k|^2 displays only the polarisation factor
|A.k|^2 x |Psi|^2 displays for all other cases.
</string>
        </property>
        <property name="currentText">
         <string>|Psi|²</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxVisibleItems">
         <number>3</number>
        </property>
        <property name="maxCount">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>|Psi|²</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>|A.k|²</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>|A.k|² x |Psi|² </string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="polarization_combobox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Choose one type of polarization.</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Toroid (p-pol)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NANO-ESCA: p-pol</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NANO-ESCA: s-pol</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NANO-ESCA: circ+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NANO-ESCA: circ-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CDAD</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ak_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Ψ</string>
        </property>
        <property name="text">
         <string>Factor:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
